software/py/vanilla_parser/pc_histogram.py

```diff
@@ -197,19 +197,17 @@ def __generate_pc_histogram(self, pc_cnt, pc_cycle):
 
         # Repeat once more for the last basic block 
         # Number of times basic block is executed
```

Comment:
Can you update the comment for the new if statment?

Reply:
I have it checking for pc_list earlier and the function now and comment that the function returns empty dirctionary if pc_list is empty